README.md

```diff
@@ -1 +1,36 @@
-# skills-ml
\ No newline at end of file
+# skills-ml
+Open Skills Project - Machine Learning
+
+This is the home for the methods and pipelines responsible for creating a skills and jobs ontology, usable by the Open Skills API.
+
+## Quick Start:
+
+### 1. Virtualenv
+skills-ml depends on python3, so create a virtual environment using a python3 executable.
+
+`virtualenv venv -p /usr/bin/python3`
+
+### 2. Airflow
+Although many of the algorithmic components can be used as pure Python classes, the main way of running this code is on Airflow. There are a number of things to get Airflow up and running, but the script below will launch Airflow with a reasonable default configuration for this repository, and even start the webserver.
+
+`sh tests/basic_deploy_skills_ml.sh`
+
+The script will prompt you to use the Airflow UI to input S3 credentials. That should be done at this stage.
+
+### 3. Configure s3 buckets and prefixes
+
+Configuration of external data sources, such as s3, is controlled via a config.yaml file in the root directory of the project. Copy example_config.yaml to example_config.yaml and modify it to match the source buckets and paths for the data you wish to use (such as job listings, or ONET extracts).
+
+### 4. Run whatever DAGs or tasks you like
+For instance, this code will test a sample ONET title extractor:
+`PYTHONPATH='.' airflow test simple_machine_learning title_extract 2016-01-01`
```

Comment:
I would make it explicit that the user is expected to be at the root directory of `skills-ml` when running this example, especially since we haven't made this library pip installable yet. Other than that it all looks great.